spec/views/articles/index.html.erb_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe 'articles/index.html.erb', type: :view do
+
+  describe 'GET #index' do
+      it 'displays a list of articles' do
```

Comment:
What are the other possible scenarios? Although this scenario reads `displays a list of articles` the test only have one article.

When implementing a listing of items usually there are three scenarios:

- There are no items
- There is only one item
- There is more than one item

Reply:
Great to have a wider breadth. This has been implemented now.